src/styles/brackets.less

```diff
@@ -511,8 +511,7 @@ a, img {
             margin-bottom: 0;
             .sprite-icon(0, 0, 13px, 13px, "images/flip-view-icons.svg");
             background-origin: content-box;
-            -webkit-transform: translateZ(0); // forces GPU mode for better filter rendering on retina
-            transform: translateZ(0); // future proofing
+            transform: translateZ(0); // forces GPU mode for better filter rendering on retina
```

Comment:
Please note that the Linux build of Brackets Shell is sadly still based on CEF 1547 (~ Chrome 29), so you still need these prefixes (and probably others, too) according to caniuse.com